### DISCUSSION TITLE
How to set the PDF Title, Author, and Subject using CLI

### DISCUSSION THREAD

**Discussion by volodymyrprokopyuk** [pdfcpu/pdfcpu#616] [Q&A] [answered]
Hi,

Is it possible to set the PDF Title, Author, and Subject using the `pdfcpu` CLI? I know that there are the `properties` and `keywords` `pdfcpu` commands, but I'm specifically interested in the above three meta fields of a PDF file.

Thank you,
Vlad

**Comment by hhrutter** [comment, marked as answer]
Hi!

This should work:
`pdfcpu properties add test.pdf 'Title = myTitle'` 
`pdfcpu info test.pdf`

**Reply by volodymyrprokopyuk** [reply]
Thank you @hhrutter it worked!

When I do `pdfcpu properties add scores/При-долине-куст-калины-oz1w.pdf 'Title = При долине куст калины'`
I get
```
pdfcpu info scores/При-долине-куст-калины-oz1w.pdf
         PDF version: 1.7
         Page count: 1
         Page size: 596.00 x 842.00 points
............................................
         Title: @8 4>;8=5 :CAB :0;8=K
         Author:
         ...
```
Does `pdfcpu` supports UTF-8 to correctly display Cyrillic? I'm using that `wezterm` that correctly works with Cyrillic.

Thank you,
Vlad

**Comment by hhrutter** [comment]
There seems to be a string encoding bug when adding properties.
Please do me a favor and [file an issue](https://github.com/pdfcpu/pdfcpu/issues/new/choose).
Thank you!

**Reply by volodymyrprokopyuk** [reply]
Hi @hhrutter,

I've created the [issue](https://github.com/pdfcpu/pdfcpu/issues/617)

Thank you,
Vlad
